recipes/chronometrist

```diff
@@ -1,6 +1,9 @@
 (chronometrist
- :repo "contrapunctus-1/chronometrist"
- :fetcher github
+ :url "https://tildegit.org/contrapunctus/chronometrist.git"
+ :fetcher git
+ :branch "production"
  :files (:defaults
          "elisp/*.el"
-         (:exclude "elisp/chronometrist-goals.el")))
+         "elisp/*.org"
+         (:exclude "elisp/chronometrist-key-values*")
+         (:exclude "elisp/chronometrist-goal*")))
```

Comment:
You have removed `elisp/chronometrist-goal*` so you don't need to exclude them anymore.

What about `elisp/chronometrist-key-values*`; are you going to create a separate repository for those as well?

Reply:
I was thinking of moving chronometrist-goal to the chronometrist repository and made this change as a preemptive measure. 🙂

Reply:
Ah, that changes things of course.

What about Steve's comment above?